presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorColumnIdentity.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.raptor;
+
+import com.facebook.presto.spi.ColumnIdentity;
+import com.facebook.presto.spi.PrestoException;
+import com.google.common.primitives.Longs;
+
+import static com.facebook.presto.spi.StandardErrorCode.GENERIC_INTERNAL_ERROR;
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.io.BaseEncoding.base16;
+import static com.google.common.primitives.Longs.fromByteArray;
+import static java.lang.String.format;
+import static java.lang.System.arraycopy;
+import static java.util.Arrays.copyOfRange;
+import static java.util.Objects.hash;
+import static java.util.Objects.requireNonNull;
+
+public class RaptorColumnIdentity
+        implements ColumnIdentity
+{
+    private static final byte CURRENT_VERSION = 1;
+
+    private final byte version;
```

Comment:
Don't store version here -- it's not needed, and it's not clear if it would be needed in the future. Since new versions might need different information, a subclass per version would be a reasonable way to go.